### ISSUE TITLE
Add dart:ui and dart:jni API docs to docs.flutter.io

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter#998]
This is probably blocked on https://github.com/flutter/flutter/issues/523 and https://github.com/dart-lang/dartdoc/issues/1067

**Comment by eseidelGoogle** [comment]
@sethladd seems most familiar with how we're going to resolve this.

Once we're unblocked from the dartdoc work, our script for generating our docs is here:
https://github.com/flutter/flutter/blob/master/travis/test.sh#L21
But for this change we'll likely need to do so from the engine repo, which will mean adding some similar code to:
https://github.com/flutter/engine/tree/master/travis

**Comment by eseidelGoogle** [comment]
Blocked on @pq's work, assigning to him for now.  We can re-assign once it's unblocked.

**Comment by eseidelGoogle** [comment]
Any update?  My guess is we're not going to complete this in time for next Tues.

**Comment by pq** [comment]
Sorry for the radio silence! There's a separate issue tracking build changes that are gating this: getting embedded libs added to the engine build (https://github.com/flutter/flutter/issues/1712); from the `analyzer` and `dartdoc` side we should be good.

**Comment by sethladd** [comment]
When @johnmccutchan 's work on _embedder.yaml finishing soon, we'll then be able to easily do this. We should be on track for Tuesday.

**Comment by sethladd** [comment]
Blocked on https://github.com/flutter/flutter/issues/2240

**Comment by devoncarew** [comment]
http://docs.flutter.io/dart-jni/dart-jni-library.html

http://docs.flutter.io/dart-ui/dart-ui-library.html
